src/lib/transaction/operations/manageData/ManageData.ts

```diff
@@ -0,0 +1,30 @@
+import ManageDataComponent from './ManageData.svelte';
+import type { Operation, Transaction } from 'stellar-sdk';
+import type { SvelteComponent } from 'svelte';
+
+export type ManageDataComponentType = {
+    component: typeof SvelteComponent;
+    props: {
+        type: string;
+        optionalSource: string | undefined;
+        defaultSource: string;
+        name: string;
+        value: Buffer | undefined;
+    };
+};
+
+export default class ManageData {
```

Comment:
This class should implement an interface that tells it to have a createOperation method, same as the createAccount one